smcroute.service.in

```diff
@@ -8,8 +8,9 @@ After=network-online.target
 Requires=network-online.target
 
 [Service]
-Type=notify
-ExecStart=@SBINDIR@/smcrouted -n -s
+Type=simple
+EnvironmentFile=@sysconfdir@/default/smcroute
+ExecStart=@SBINDIR@/smcrouted -n -s $SMCROUTED_OPTS
 ExecReload=@SBINDIR@/smcroutectl restart
```

Comment:
This is a cool idea. Do you know if systemd requires the file to exist or can it be optional on user installs?

Reply:
Thanks for the reminder. I should use "-" before the directory in case the file doesn't exist.

Reply:
Ah, so that's the syntax. Thanks!